platform/features/plot/src/PlotOptionsController.js

```diff
@@ -0,0 +1,145 @@
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+/*global define*/
+
+define(
+    ['./PlotOptionsForm'],
+    function (PlotOptionsForm) {
+        "use strict";
+
+        /**
+         * Notes on implementation of plot options
+         *
+         * Multiple y-axes will have to be handled with multiple forms as
+         * they will need to be stored on distinct model object
+         *
+         * Likewise plot series options per-child will need to be separate
+         * forms.
+         */
+
+        /**
+         * The LayoutController is responsible for supporting the
+         * Layout view. It arranges frames according to saved configuration
+         * and provides methods for updating these based on mouse
+         * movement.
+         * @memberof platform/features/plot
+         * @constructor
+         * @param {Scope} $scope the controller's Angular scope
+         */
+        function PlotOptionsController($scope, topic) {
+
+            var self = this,
+                domainObject = $scope.domainObject,
+                composition,
+                mutationListener,
+                formListener,
+                configuration = domainObject.getModel().configuration || {};
+
+            this.plotOptionsForm = new PlotOptionsForm(topic);
+
+            /*
+             * Determine whether the changes to the model that triggered a
+             * mutation event were purely compositional.
+             */
+            function hasCompositionChanged(oldComposition, newComposition){
+                // Framed slightly strangely, but the boolean logic is
+                // easier to follow for the unchanged case.
+                var isUnchanged = oldComposition === newComposition ||
+                        (
+                            oldComposition.length === newComposition.length &&
+                            oldComposition.every( function (currentValue, index) {
+                                return newComposition[index] && currentValue === newComposition[index];
+                            })
+                        );
+                return !isUnchanged;
+            }
+
+            /*
+             Default the plot options model
+             */
+            function defaultConfiguration() {
+                configuration.plot = configuration.plot || {};
+                configuration.plot.xAxis = configuration.plot.xAxis || {};
+                configuration.plot.yAxis = configuration.plot.yAxis || {}; // y-axes will be associative array keyed on axis key
+                configuration.plot.series = configuration.plot.series || {}; // series will be associative array keyed on sub-object id
```

Comment:
As per https://github.com/nasa/openmctweb/pull/645#issuecomment-180021486 I was hoping to leave this for the "wiring up" stage with the actual plot. It complicates things in the forms a bit, and wanted to get this in before code freeze to spread out the code reviews a little. Let me see if I can get something in today.